shared/chat-sdk/client-api/chat-room/_manage-chatroom-attributes.mdx

```diff
@@ -2,19 +2,20 @@ import * as data from '@site/data/variables';
 import Understand from '@docs/shared/chat-sdk/client-api/chat-room/manage-chatroom-attributes/understand/index.mdx';
 import ProjectImplement from '@docs/shared/chat-sdk/client-api/chat-room/manage-chatroom-attributes/project-implementation/index.mdx';
 
-Chat rooms enable real-time messaging among multiple users.
+Chat room attributes consist of basic attributes (such as room subject, room description, and room announcement) and custom attributes. When basic attributes cannot satisfy the business requirements, users can add custom attributes that are synchronized with all chat room members.
```

Comment:
1. Please drop the parenthesis and try to avoid them in docs. 
2. Are "users" and "you" the same people in this article? If so, can we stick to one? (I would prefer users for people actually logging into the app)